frontend/src/pages/register/index.tsx

```diff
@@ -1,100 +1,30 @@
 import React, { useState } from 'react'
-import { Form, Header } from '../../components'
-import { useAuthService } from '../../hooks/useAuth'
+import { Header } from '../../components'
 import {
   Wrapper,
   Container,
   Feature,
 } from '../../components/layout/layout.styles'
-import {
-  Title,
-  SubTitle,
-  AccentText,
-  Link,
-} from '../../components/text/text-styles'
+import { Title, SubTitle } from '../../components/text/text-styles'
 import RegisterForm from './form'
 
 const Register = () => {
-  const [showPassword, setShowPassword] = useState(false)
-  const [showConfirmPassword, setShowConfirmPassword] = useState(false)
-  const [email, setEmail] = useState<string>('')
-  const [password, setPassword] = useState<string>('')
-  const [confirmPassword, setConfirmPassword] = useState<string>('')
-  const [firstName, setFirstName] = useState<string>('')
-  const [lastName, setLastName] = useState<string>('')
-  const [error, setError] = useState()
+  const [error, setError] = useState<string>('')
   const [loading, setLoading] = useState(false)
+  const [formData, setFormData] = useState()
```

Comment:
You probably don't need formData in the state here. Just pass it to handleSubmit directly from the form